stories/icon/data.tsx

```diff
@@ -47,7 +210,240 @@ export const ICONS: JSX.Element[] = [
     <ChevronLeftIcon key="ChevronLeftIcon" />,
     <ChevronRightIcon key="ChevronRightIcon" />,
     <ChevronUpIcon key="ChevronUpIcon" />,
-    // Chevron Line
+    // Chevron
     <ChevronLineLeftIcon key="ChevronLineLeftIcon" />,
     <ChevronLineRightIcon key="ChevronLineRightIcon" />,
+    <Chevron2LeftIcon key="Chevron2LeftIcon" />,
+    <Chevron2RightIcon key="Chevron2RightIcon" />,
+    // Caret
+    <CaretLeftIcon key="CaretLeftIcon" />,
+    <CaretRightIcon key="CaretRightIcon" />,
+    <CaretUpIcon key="CaretUpIcon" />,
+    <CaretDownIcon key="CaretDownIcon" />,
+    // Triangle
+    <TriangleForwardIcon key="TriangleForwardIcon" />,
+    <TriangleForwardFillIcon key="TriangleForwardFillIcon" />,
+    // Ellipsis
+    <EllipsisHorizontalIcon key="EllipsisHorizontalIcon" />,
+    <EllipsisVerticalIcon key="EllipsisVerticalIcon" />,
+    // Plus
+    <PlusIcon key="PlusIcon" />,
+    <PlusCircleIcon key="PlusCircleIcon" />,
+    <PlusCircleFillIcon key="PlusCircleFillIcon" />,
+    // Minus
+    <MinusIcon key="MinusIcon" />,
+    <MinusCircleIcon key="MinusCircleIcon" />,
+    <MinusCircleFillIcon key="MinusCircleFillIcon" />,
+    // Cross
+    <CrossIcon key="CrossIcon" />,
```

Comment:
Sorry i believe i did not mention this clearly. But could you help me arrange them alphabetically by their group?